CMakeLists.txt

```
@@ -275,26 +311,27 @@ if(enable-glib)
 endif()
 
 add_definitions(-DMALIIT_FRAMEWORK_USE_INTERNAL_API
-                -DMALIIT_PLUGINS_DATA_DIR="${CMAKE_INSTALL_FULL_DATADIR}/maliit/plugins"
-                -DMALIIT_EXTENSIONS_DIR="${CMAKE_INSTALL_FULL_DATADIR}/maliit-framework/extensions"
+                -DMALIIT_PLUGINS_DATA_DIR="${CMAKE_INSTALL_FULL_DATADIR}/maliit${MALIIT_LIB_SUFFIX}/plugins"
+                -DMALIIT_EXTENSIONS_DIR="${CMAKE_INSTALL_FULL_DATADIR}/maliit${MALIIT_LIB_SUFFIX}-framework/extensions"
                 -DMALIIT_CONFIG_ROOT="/maliit/"
-                -DMALIIT_PLUGINS_DIR="${CMAKE_INSTALL_FULL_LIBDIR}/maliit/plugins"
-                -DMALIIT_DEFAULT_HW_PLUGIN="libmaliit-keyboard-plugin.so"
+                -DMALIIT_PLUGINS_DIR="${CMAKE_INSTALL_FULL_LIBDIR}/maliit${MALIIT_LIB_SUFFIX}/plugins"
+                -DMALIIT_DEFAULT_HW_PLUGIN="libmaliit${MALIIT_LIB_SUFFIX}-keyboard-plugin.so"
```

```suggestion
                -DMALIIT_DEFAULT_HW_PLUGIN="libmaliit-keyboard-plugin.so"
```
same 2 lines below too, otherwise `maliit6-server` fails to load the plugin with e.g. the following since it's actually installed in `/usr/lib/maliit6/plugins/libmaliit-keyboard-plugin.so`:
```
org.maliit.framework: bool MIMPluginManagerPrivate::loadPlugin(const QDir &, const QString &) Error loading plugin from "/usr/lib/maliit6/plugins/libmaliit6-keyboard-plugin.so" "The shared library was not found."
```

---

Thanks a lot for the review! I just pushed the changes.
How to handle the dbus service file with respect to coinstallability is tricky though because this is the single file that produces a file clash between Qt5 and Qt6 version. Not sure if it is OK to just ignore it and let downstream figure out which one of them should be installed?